examples/Get OAuth2 access token from AAD using client id and certificate using key vualt manage identity.xml

```diff
@@ -0,0 +1,127 @@
+﻿<!-- The policy defined in this file provides an example of using OAuth2 for authorization between the gateway and a backend using Certificate from Key Vault -->
+<!-- It shows how to obtain an access token from AAD, cache it for a configurable amount of time and forward it to the backend. -->
+
+
+<!-- Send request to AAD to obtain a bearer token -->
+<!-- Parameters: authorizationServer - format https://login.windows.net/TENANT-GUID/oauth2/token -->
+<!-- Parameters: scope - a URI encoded scope value -->
+<!-- Parameters: clientId - an id obtained during app registration -->
+<!-- Parameters: KeyVaultName - Key Vault name where we need to get certificate -->
+<!-- Parameters: CertName - Certificate name in Key Vault -->
+<!-- Parameters: cachetime - Cache time in seconds for token -->
+<!-- Copy the following snippet into the inbound section. -->
+
+<policies>
+  <inbound>
+    <cache-lookup-value key="@("bearerToken")" variable-name="bearerToken" />

```

Comment:
does "key" have to be an expression here?

Reply:
we can remove this,  I was getting token from root policy under APIM